db/schema.rb

```diff
@@ -10,65 +10,69 @@
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_08_22_141252) do
+ActiveRecord::Schema.define(version: 2019_08_31_100615) do
 
-  create_table "certificates", force: :cascade do |t|
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "pgcrypto"
+  enable_extension "plpgsql"
+
+  create_table "certificates", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
     t.string "value"
     t.string "usage"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.integer "component_id"
+    t.uuid "component_id"
     t.boolean "enabled", default: true
     t.string "component_type"
     t.index ["component_id"], name: "index_certificates_on_component_id"
   end
 
-  create_table "events", force: :cascade do |t|
+  create_table "events", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
     t.string "type", null: false
     t.json "data"
     t.string "aggregate_type"
-    t.integer "aggregate_id"
+    t.uuid "aggregate_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.string "user_id"
     t.index ["aggregate_type", "aggregate_id"], name: "index_events_on_aggregate_type_and_aggregate_id"
   end
 
-  create_table "msa_components", force: :cascade do |t|
+  create_table "msa_components", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
     t.string "name"
     t.string "component_type"
-    t.integer "encryption_certificate_id"
+    t.uuid "encryption_certificate_id"
     t.string "entity_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.integer "team_id"
-    t.string "environment", null: false
+    t.uuid "team_id"
+    t.string "environment"
     t.index ["team_id"], name: "index_msa_components_on_team_id"
   end
 
-  create_table "services", force: :cascade do |t|
+  create_table "services", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
     t.string "entity_id", null: false
     t.string "name"
-    t.integer "sp_component_id"
-    t.integer "msa_component_id"
+    t.uuid "sp_component_id"
+    t.uuid "msa_component_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.index ["entity_id"], name: "index_services_on_entity_id", unique: true
   end
 
-  create_table "sp_components", force: :cascade do |t|
+  create_table "sp_components", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
     t.string "name"
     t.string "component_type"
-    t.integer "encryption_certificate_id"
+    t.uuid "encryption_certificate_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.boolean "vsp", default: false
-    t.integer "team_id"
-    t.string "environment", null: false
+    t.uuid "team_id"
+    t.string "environment"
```

Comment:
How come we're losing `null: false` here?

Reply:
Thanks for spotting this; i'll update this